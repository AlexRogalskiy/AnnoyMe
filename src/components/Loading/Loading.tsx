import React from "react";
import { Text, View } from "react-native";
import styles from "./styles";

/** Loading indicator, mostly used with Suspend. */
const Loading = () => (
  <View style={styles.container}>
    <Text style={styles.text}>Loading...</Text>
  </View>
);

export default Loading;
