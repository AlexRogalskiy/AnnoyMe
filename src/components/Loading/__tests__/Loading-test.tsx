import React from "react";
import renderer from "react-test-renderer";
import Loading from "../Loading";

it("Loading shows loading message", () => {
  const tree = renderer.create(<Loading />).toJSON();

  expect(tree).toMatchSnapshot("Loading-shows-message");
});
