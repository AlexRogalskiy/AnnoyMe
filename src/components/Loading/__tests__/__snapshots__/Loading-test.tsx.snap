// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading shows loading message: Loading-shows-message 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 14,
        "padding": 20,
      }
    }
  >
    Loading...
  </Text>
</View>
`;
